ts/Extensions/Breadcrumbs.ts

```diff
@@ -1103,6 +1135,7 @@ namespace Breadcrumbs {
         format?: string;
         formatter?: BreadcrumbsButtonsFormatter;
         relativeTo?: ButtonRelativeToValue;
+        rtl?: boolean;
```

Comment:
I think it would be better to make it mandatory and `false` by default. Since only `true` and `false` make any sense.